login.js

```diff
@@ -0,0 +1,52 @@
+function validateForm(){
+    let usernameProgress = 0;
+    let passwordProgress = 0;
+    let currentIndex = 0;
+
+    let usernameField = document.forms["loginForm"]["username"].value;
+    let passwordField = document.forms["loginForm"]["password"].value;
+
+    let notEmpty = () => {
+        passwordField === "" ? alert("You must enter a password.") : passwordProgress += 50;
```

Comment:
Oh also, don't do this ` : passwordProgress += 50;`. Don't modify variables in a ternary.